core/src/actors/blocks_manager/messages.rs

```diff
@@ -0,0 +1,16 @@
+use actix::Message;
+
+use std::io;
+
+use witnet_data_structures::chain::CheckpointBeacon;
+
+/// Message to obtain the highest block checkpoint managed by the `BlocksManager`
+/// actor.
+pub struct GetHighestBlockCheckpointBeacon;
+
+/// Result of the GetHighestBlockCheckpoint message handling
+pub type CheckpointBeaconResult = Result<CheckpointBeacon, io::Error>;
```

Comment:
We can define our `Result` and `Error`s for `BlocksManager` in the `witnet_chain` crate, where the `BlocksManager` library will be located.

For reference, look at how `witnet_storage` defines its own `Result` and `Error`s:
https://github.com/witnet/witnet-rust/blob/master/storage/src/error.rs

Reply:
Done!!